src/main/java/com/dumbdogdiner/stickyapi/common/util/StringUtil.java

```diff
@@ -288,4 +288,12 @@ public static UUID hyphenateUUID(@NotNull String uuid) {
             return UUID.fromString(uuid);
         }
     }
+
+    public static String unhyphenateUUID(@NotNull UUID uuid){
+        return unhyphenate(uuid.toString());
+    }
+
+    public static String unhyphenate(@NotNull String str){
+        return str.replace("-","");
+    }
```

Comment:
`unhyphenate` to `dehyphenate`

Reply:
Hyphinating UUIDs is in StringUtil, so I added a dehyphenate to it

Reply:
Hmm, I guess it does make sense here, since it's returning a string. But why not call `UUID#toString()` instead of making a method for it

Reply:
Because if we get a UUID from Mojang, its an unhyphinated string, and we want a UUID object